elyra/pipeline/processor_kfp.py

```diff
@@ -99,7 +100,10 @@ def process(self, pipeline):
                 else:
                     raise lve
 
-            run = client.run_pipeline(experiment_id=client.create_experiment(pipeline_name).id,
+            experiment = client.create_experiment(name=pipeline_name,
+                                                  namespace=user_namespace)
```

Comment:
This is not a required name, I believe @kevin-bates mentioned before that there is a need to handle `defaults` as a special case here as an empty string wouldn't sufice?

Reply:
Never mind, I looked in the code and we are ok.